integration/simulation/transaction_injector.go

```diff
@@ -232,12 +233,49 @@ func (ti *TransactionInjector) issueRandomTransfers() {
 		// todo - retrieve receipt
 
 		go ti.TxTracker.trackTransferL2Tx(signedTx)
-		sleepRndBtw(ti.avgBlockDuration/4, ti.avgBlockDuration)
+		sleepRndBtw(ti.avgBlockDuration/10, ti.avgBlockDuration/4)
 	}
 }
 
 // issueRandomDeposits creates and issues a number of transactions proportional to the simulation time, such that they can be processed
 func (ti *TransactionInjector) issueRandomDeposits() {
+	fromWalletHoc := ti.wallets.Tokens[testcommon.HOC].L2Owner
```

Comment:
when the bridge stuff was refactored, we lost the deposits which meant that no l2 erc20 transfers succeeded